tsl/src/init.c

```diff
@@ -223,8 +223,24 @@ ts_module_init(PG_FUNCTION_ARGS)
 
 	/* Register a cleanup function to be called when the backend exits */
 	if (register_proc_exit)
+	{
 		on_proc_exit(ts_module_cleanup_on_pg_exit, 0);
 
+		bloom1_column_prefix = default_bloom1_column_prefix;
+#ifndef NDEBUG
+		DefineCustomStringVariable(MAKE_EXTOPTION("debug_bloom1_column_prefix"),
+								   "bloom filter column prefix",
+								   NULL,
+								   (char **) &bloom1_column_prefix,
+								   default_bloom1_column_prefix,
+								   PGC_USERSET,
+								   0,
+								   NULL,
+								   NULL,
+								   NULL);
+#endif
```

Comment:
Why is this tied to register_proc_exit and NDEBUG?

Reply:
This is only for tests, I need to create some way to set the bloom filter column prefix to create the old version.

I can't register this in the Apache part because the default value is dependent on the compilation settings of the TSL part. There's no existing place for this in the TSL part, this place seems to be called at the right time.

Reply:
I extended the comments. I think I'll keep it as a read-only GUC in the release builds as well, to allow for some observability as we discussed before. The user will be able to query it to check which hashing schema is currently in use.